assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["PIE*", "CYY", "DEEDS", "WORPP", "DHP"],
+                ["RSW*", "CYY"],
+                ["SRQ*", "CYY"]
+            ]
         },
         "DVALL2": {
             "icao": "DVALL2",
-            "name": "Dvall Two",
+            "name": "DaVall Two",
             "entryPoints": {
-                "EYW": ["EYW", "CARNU", "DVALL"],
-                "MTH": ["MTH", "DVALL"]
+                "EYW": ["EYW", "CARNU"],
+                "MTH": ["MTH"]
             },
-            "body": [["WEVER", "A100|S250"], "FAMIN", "DHP"],
-            "draw": [["EYW*","CARNU","DVALL"],["MTH*","DVALL"],["DVALL","WEVER","FAMIN","DHP"]]
+            "body": ["DVALL", "WEVER", "FAMIN", "DHP"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["EYW*", "CARNU", "DVALL", "WEVER", "FAMIN", "DHP"],
+                ["MTH*", "DVALL"]
+            ]
         },
         "FLIPR4": {
             "icao": "FLIPR4",
             "name": "Flipper Four",
             "entryPoints": {
-                "ZFP": ["ZFP", "LUCSS", "FOWEE"],
-                "ZQA": ["ZQA", "NICKO", "TINKY", "LUCSS", "FOWEE"],
-                "HODGY": ["HODGY", "JAYEE", "PEKRE", "TINKY", "LUCSS", "FOWEE"],
-                "URSUS": [["URSUS", "S280"], "ZOLLA", "FOWEE"]
+                "HODGY": ["HODGY", "JAYEE", "PEKRE", "TINKY", "LUCSS"],
+                "URSUS": ["URSUS", "ZOLLA"],
+                "ZFP": ["ZFP", "LUCSS"],
+                "ZQA": ["ZQA", "NICKO", "TINKY", "LUCSS"]
             },
-            "body": [["FLIPR", "A120|S280"], "LUVLY", "LECIT", "JODPO"],
-            "draw": [["ZFP*","LUCSS","FOWEE"],["ZQA*","NICKO","TINKY","LUCSS","FOWEE"],["HODGY*","JAYEE","PEKRE","TINKY","LUCSS","FOWEE"],["URSUS*","ZOLLA","FOWEE"],["FOWEE","FLIPR","LUVLY","LECIT","JODPO"]]
+            "body": ["FOWEE", "FLIPR", "LUVLY", "LECIT", "JODPO"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["HODGY*", "JAYEE", "PEKRE", "TINKY", "LUCSS", "FOWEE", "FLIPR", "LUVLY", "LECIT", "JODPO"],
+                ["URSUS*", "ZOLLA", "FOWEE"],
+                ["ZFP*", "LUCSS", "FOWEE"],
+                ["ZQA*", "NICKO", "TINKY", "LUCSS"]
+            ]
         },
         "FOWEE8": {
```

Comment:
FOWEE8 is now FOWEE9

Reply:
reverified and updated all SIDs/STARs.

done